pkgs/applications/editors/jetbrains/default.nix

```
@@ -93,8 +93,8 @@ let
         chmod +x $out/goland*/plugins/go/lib/dlv/linux/dlv
 
         # fortify source breaks build since delve compiles with -O0
-        wrapProgram $out/goland*/plugins/go/lib/dlv/linux/dlv \
-          --prefix disableHardening " " fortify
+        wrapProgram $out/bin/goland \
+          --set CGO_CPPFLAGS "-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0"
```

Would be prefix not better here, so that users still have a chance to expand this?

---

I'm not a Nix expert by any means, but wouldn't prefix add flags to the `goland` binary when launched instead of setting the `CGO_CPPFLAGS` environment variable like we want to do here?

---

If you do this:

```suggestion
          --prefix CGO_CPPFLAGS " " "-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0"
```

Than users could set i.e. `export CGO_CPPFLAGS="-DFOO=1"` and it would get append to the fortify flags.

---

Fair enough. Let's go with that :+1: